src/bin/cql-stress-scylla-bench/gocompat/strconv.rs

```diff
@@ -234,12 +234,14 @@ pub fn format_duration(d: Duration) -> String {
 
         if first_digit >= 1_000_000 {
             let prec = 8 - first_offset;
-            write!(s, "{:.*}ms", prec, total_nanos as f64 / 1_000_000.0).unwrap();
+            let ms = total_nanos as f64 / 1_000_000.0;
+            write!(s, "{ms:.prec$}ms").unwrap();
```

Comment:
I don't fully understand how `prec` was used...
@wprzytula is this equivalent?

Reply:
It would be good to leave it like it was before and write unit tests for it and this change in seperate PR even thou CI is failing

Reply:
Yes, although I needed to read [this](https://doc.rust-lang.org/std/fmt/#width) carefully to get convinced.